packages/source-iotsitewise/src/alarms/iotevents/constants.ts

```diff
@@ -0,0 +1,74 @@
+import { COMPARISON_OPERATOR, StatusIcon } from '@synchro-charts/core';
+import { ComparisonOperator } from '@aws-sdk/client-iot-events';
+
+export const SOURCE = 'iotevents';
+
+export const ALARM_STATUS = {
```

Comment:
Should these be an enum or a type? We wouldn't need to use Object.keys to interact with it then.

Reply:
I did try with enums at first but it turned out to be more of a hassle than using Object.keys. There are places in the code where I need to dynamically access the values 
https://github.com/awslabs/iot-app-kit/pull/248/files#diff-daee7f19efbf6c86e08afacea5aad0ada75cdba9b6bd4853862bc316d6afe611R10